client/src/components/Ticketing/ticketingmanager/showings/ShowingUpdated/EventPageV2.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import {EventShowingsContainer} from './EventShowingsContainer';
+import {EventGeneralContainer} from './EventGeneralContainer';
+import {useEvent} from './EventProvider';
+import {LoadingScreen} from '../../../mainpage/LoadingScreen';
+import Udash_nav from '../../udash_navbar';
+import PopUp from '../../../Pop-up';
+
+export const EventPageV2 = () => {
+  const {
+    eventID,
+    loading,
+    token,
+    showPopUp,
+    setShowPopUp,
+    message,
+    success,
+    title,
+  } = useEvent();
+
+  if (loading || token === '' || eventID === undefined) {
+    return <LoadingScreen />;
+  } else {
+    return (
+      <div className={'flex flex-row'}>
+        <Udash_nav />
+        <div className='w-full h-screen overflow-x-hidden absolute bg-gray-200'>
+          {showPopUp ? (
```

Comment:
Same as above, also could you remove the `<> and </>` in this case since these only wraps one element